### ISSUE TITLE
Relative path failure

### ISSUE THREAD

**Issue by FLefb** [box/boxcli#103]
Hi.

When I use:
_box configure environments add "D:\MyWorkingFolder\bin\myConfig.json" --set-as-current_
the session opens correctly.

But when using:
_cd \MyWorkingFolder\bin
box configure environments add "myConfig.json" --set-as-current_
box app looks for the file
_\D:MyWorkingFolder\bin\myConfig.json_
and obviously fails.

It seems there is an issue with path parsing/generation.

**Comment by mattwiller** [comment]
@FLefb Thanks for letting us know!  I was able to reproduce this issue; we're currently working on a new version of the CLI and will make sure that this gets resolved in that release.

For now, you should be able to work around the issue by prefixing your path with `.\`, like this:

```
cd \MyWorkingFolder\bin
box configure environments add ".\myConfig.json" --set-as-current
```

Please let me know if that doesn't work for you!

**Comment by mattwiller** [comment]
@FLefb We just released v2.0.0 of the CLI, which should resolve this issue.  You can find the new installers at https://github.com/box/boxcli/releases/tag/v2.0.0 — please give it a try and let us know if you run into any other issues!
